pkg/aks/update.go

```diff
@@ -108,14 +108,23 @@ func updateCluster(desiredCluster armcontainerservice.ManagedCluster, actualClus
 		}
 	}
 
+	// If the cluster is imported, we may not have the authorized IP ranges set on the spec.
```

Comment:
Q: is there a case where  authorized IP ranges are set on the spec and if so what happens in that case?

Reply:
For imported clusters it is not possible, always this setting will be overwritten with downstream cluster